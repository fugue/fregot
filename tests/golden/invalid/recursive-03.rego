package recursive_03

foo {
    data.recursive_03.bar
}

bar {
    qux
}

qux {
    data.recursive_03.foo
}

