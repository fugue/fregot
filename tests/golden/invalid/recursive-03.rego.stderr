fregot (recursion check error):
  "recursive-03.rego" (line 3, column 1):
  rules are recursive:

    3| foo {
       ^^^

  These rules are mutually recursive:
    foo
    bar
    qux

  "recursive-03.rego" (line 7, column 1):
  rules are recursive:

    7| bar {
       ^^^

  These rules are mutually recursive:
    foo
    bar
    qux

  "recursive-03.rego" (line 11, column 1):
  rules are recursive:

    11| qux {
        ^^^

  These rules are mutually recursive:
    foo
    bar
    qux
